src/modules/kvs/kvs.c

```diff
@@ -2882,6 +2975,8 @@ int mod_main (flux_t *h, int argc, char **argv)
         flux_log_error (h, "error creating KVS context");
         goto done;
     }
+    if (process_config (ctx) < 0)
```

Comment:
Maybe for a future PR: support `kvs.config-reload` so this tunable can be changed on the fly.